### ISSUE TITLE
abap: security

### ISSUE THREAD

**Issue by larshp** [larshp/WABAP#21]
authorizations and injection stuff

**Comment by chetan2691** [comment]
Not a clear question

**Comment by larshp** [comment]
no, its mainly just notes for myself to remember what is missing

if you like to contribute, I can elaborate?
